src/main/webapp/app/exercise/shared/entities/exercise/exercise.model.ts

```diff
@@ -149,6 +149,7 @@ export abstract class Exercise implements BaseEntity {
     protected constructor(type: ExerciseType) {
         this.type = type;
         this.bonusPoints = 0; // default value
+        this.maxPoints = 1; // default value
```

Comment:
That might be a huge change when creating new exercises. It could be that instructors simply overlook it.